net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="A small dynamic DNS client"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers \
+support and a high flexibility."
+HOMEPAGE="https://yaddns.github.io/"
+COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/yaddns/yaddns/archive/yaddns-$portVersion.tar.gz"
+CHECKSUM_SHA256="23b527b7f71e1449746f1e39cbc3a074d58bd261a33399fb4e2d90eb63149059"
+SOURCE_DIR="yaddns-yaddns-$portVersion"
+PATCHES="yaddns-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]
+then
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
+
+PROVIDES="
+	yaddns$secondaryArchSuffix = $portVersion
+	cmd:yaddns$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autom4te
+	cmd:automake
+	cmd:autoreconf
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:makeinfo
+	"
+
+defineDebugInfoPackage yaddns$secondaryArchSuffix \
+	$commandBinDir/yaddns
+
+PATCH()
+{
+	sed -i -e "s|/etc\(/yaddns\.conf\)|`finddir B_SYSTEM_SETTINGS_DIRECTORY`\1|;" \
+		doc/yaddns.1
+}
+
+BUILD()
+{
+	./autogen.sh
+	LIBS="-Wl,--as-needed -lbnetapi -lnetwork" runConfigure \
+		--omit-dirs binDir ./configure --bindir $commandBinDir
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	install -d $docDir
+	install AUTHORS $docDir
+	install etc/yaddns.conf $docDir/yaddns.conf.example
+}
+
+TEST()
+{
+	make check
```

Comment:
```diff
 TEST()
 {
-	make check
 	LIBS="-Wl,--as-needed -lbnetapi -lnetwork" make check
 }
```
BTW, in your last commit, the comment is:
```
update recipe

o Install AUTHORS and yaddns.conf to docDir.
o Fix test function.
```
Did you really mean "Fix test function" or did you already fix the patchset itself?

(I'm just asking in case you did fix it but forgot to commit the fix. There is no urgency, and you should not hesitate to postpone the fix until you have completed as many other tasks you would like to claim before the end of the GCI.)

Reply:
@fbrosson Oh thanks. I really need to double check before committing.

> Did you really mean "Fix test function" or did you already fix the patchset itself?

I meant the __`TEST()`__ function only, not the patchset, as the commit message only says "update recipe". I haven't fixed the patchset yet.